src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -270,6 +270,28 @@ public void testSwitch() {
         switchModifications(uriString + "&variantId=" + NetworkCreation.VARIANT_ID, "break1Variant", "notFound", "disc1Variant", "break2Variant", Set.of("s1Variant"), Set.of("s2Variant"), 8);
     }
 
+    @Test
+    public void testDeleteModification() {
```

Comment:
Maybe add some test cases :
* to delete an non-existing modification
* to delete an existing modification but from an non-existing group 
What about error managment ?

Reply:
done